frontend/Components/ListingCard/ListingCard.module.css

```
@@ -0,0 +1,41 @@
+.cardContainer {
```

Ya que usamos CSS modules, esto se puede llamar:

```suggestion
.container {
```